src/app/messages/en/attributionText.ts

```diff
@@ -0,0 +1,59 @@
+export default `
+<p>
+  Want to cite, share, or modify this book? This book is {bookLicenseName}
+  {bookLicenseVersion} and you must attribute OpenStax.
+</p>
+
+<strong>Attribution information</strong>
+<ul>
+  <li>
+    If you are redistributing all or part of this book in a print format,
+    then you must include on every physical page the following attribution:
+    Access for free at https://openstax.org{introPageUrl}
```

Comment:
This might not be in the mockup but could these URL's be clickable?
```suggestion
    <blockquote>
      Access for free at <a href="https://openstax.org{introPageUrl}">https://openstax.org{introPageUrl}</a>
    </blockquote>
```